tests/unit/amo/components/TestCategories.js

```diff
@@ -129,21 +132,27 @@ describe(__filename, () => {
   });
 
   it('renders loading text when loading', () => {
-    const root = render({
-      addonType: ADDON_TYPE_EXTENSION,
-      loading: true,
-    });
+    store.dispatch(
+      fetchCategories({
+        errorHandlerId: createStubErrorHandler().id,
+      }),
+    );
+
+    const root = render({ addonType: ADDON_TYPE_EXTENSION });
```

Comment:
no `loading` now?

Reply:
Passing `loading` as a property was never really the right way to do it. Ideally, if you want a component to be in a loading state, you dispatch an action that will put it into that state (which is what I've done above with `store.dispatch(fetchCategories(...))`). Most of this patch is either removing tests that are no longer needed, or replacing tests that explicitly tested `mapStateToProps` with tests that test the functionality of `mapStateToProps` without explicitly calling it. This is an example of the latter. In this case it is `mapStateToProps` that sets `loading`, which is based on the current Redux state, which I set by dispatching `fetchCategories`.